<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:1,
            stop:0 #e3f2fd,
            stop:1 #bbdefb
        );</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>228</width>
      <height>349</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
        font-size: 20px;
        font-weight: bold;
        color: #333;
        background-color: transparent;
        text-align: center;
        padding: 10px;
        border-radius: 8px;
        transition: color 0.3s ease-in-out;
    }

    QLabel:hover {
        color: #42a5f5;
    }

    QLabel:focus {
        color: #1e88e5;
    }
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true"> QGroupBox {
        border: 1px solid rgba(255, 255, 255, 0.3);
        border-radius: 16px;
        background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:1,
            stop:0 rgba(255, 255, 255, 0.25),
            stop:1 rgba(255, 255, 255, 0.05)
        );
        backdrop-filter: blur(10px); /* не поддерживается напрямую, но можно имитировать */
        padding: 15px;
        margin-top: 30px;
    }

    QGroupBox:title {
        subcontrol-origin: margin;
        subcontrol-position: top left;
        padding: 0 12px;
        color: #2c3e50;
        font-size: 18px;
        font-weight: 600;
        background-color: transparent;
    }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="UsernameTextLine">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 8px 12px;
        font-size: 16px;
        background-color: white;
	    color:#5203fc;
    }

    QLineEdit:focus {
        border: 2px solid #4f46e5;
        background-color: #f9f9ff;
    }</string>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PasswordTextLine">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 8px 12px;
        font-size: 16px;
        background-color: white;
	    color:#5203fc;
    }

    QLineEdit:focus {
        border: 2px solid #4f46e5;
        background-color: #f9f9ff;
    }</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
        background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:1,
            stop:0 #42a5f5,
            stop:1 #1e88e5
        );
        color: white;
        border: none;
        border-radius: 10px;
        padding: 8px 16px;
        font-size: 14px;
    }

    QPushButton:hover {
        background-color: qlineargradient(
            x1:0, y1:0, x2:1, y2:1,
            stop:0 #64b5f6,
            stop:1 #2196f3
        );
    }

    QPushButton:pressed {
        background-color: #1565c0;
    }</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="styleSheet">
             <string notr="true">QRadioButton {
        background-color: transparent;
        border: 2px solid #bbb;
        border-radius: 12px;
        padding: 8px;
        font-size: 16px;
        color: #444;
    }

    QRadioButton::indicator {
        width: 20px;
        height: 20px;
        border: 2px solid #bbb;
        border-radius: 10px;
        background-color: white;
        margin: 5px;
    }

    QRadioButton::indicator:checked {
        background-color: #42a5f5;
        border: 2px solid #42a5f5;
    }

    QRadioButton::indicator:unchecked {
        background-color: white;
        border: 2px solid #bbb;
    }

    QRadioButton:hover {
        color: #0069c0;
    }

    QRadioButton:checked:hover {
        color: #ffffff;
    }</string>
            </property>
            <property name="text">
             <string>Registration</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
